packages/amplify-migration-template-gen/src/migration-readme-generator.ts

```diff
@@ -94,7 +94,8 @@ aws cloudformation update-stack \\
  --stack-name ${this.gen2CategoryStackName} \\
  --template-body file://${step2FileNamePath} \\
  --parameters '${paramsString}' \\
- --capabilities CAPABILITY_NAMED_IAM
+ --capabilities CAPABILITY_NAMED_IAM \\
+ --tags '[]'
```

Comment:
Empty stack level tags is a temporary workaround for stack refactor operation. It will be generated once the final Gen2 deploy runs post refactor.